evap/staff/templates/staff_semester_view.html

```diff
@@ -11,7 +11,8 @@ <h3>
         {% if request.user.is_manager %}
             &nbsp;
             <a href="{% url 'staff:semester_edit' semester.id %}" class="btn btn-sm btn-secondary">{% trans 'Edit' %}</a>
-            <button type="button" id="deleteSemesterButton"{% if not semester.can_be_deleted_by_manager %} disabled{% endif %} onclick="deleteSemesterModalShow({{ semester.id }}, '{{ semester.name|escapejs }}');" class="btn btn-sm btn-danger">{% trans 'Delete' %}</button>
+            <button onClick="makeActiveSemesterModalShow({{ semester.id }})" class="btn btn-sm btn-danger" {% if semester.is_active %}disabled{% endif %} id="makeActiveSemesterButton" type="button">{% trans 'Make active' %}</button>
+            <button type="button" id="deleteSemesterButton"{% if not semester.can_be_deleted_by_manager or semester.is_active %} disabled{% endif %} onclick="deleteSemesterModalShow({{ semester.id }}, '{{ semester.name|escapejs }}');" class="btn btn-sm btn-danger">{% trans 'Delete' %}</button>
```

Comment:
So you made `is_active == False` a condition for a manager to delete a semester? This check should then be done in `can_be_deleted_by_manager`.